src/tasks/flare.py

```diff
@@ -1080,3 +1080,57 @@ def process_results(self, doc, results):
         pred = process_zhtext(results[0], text)
 
         return {"entity_f1": (pred, doc["label"], results[0])}
+
+
+class ZHFPB(Classification):
+    DATASET_PATH = "ChanceFocus/flare-zh-fpb"
+
+
+class ZHFIQASA(Classification):
+    DATASET_PATH = "ChanceFocus/flare-zh-fiqasa"
+
+
+class ZHHeadlines(Headlines):
+    DATASET_PATH = "ChanceFocus/flare-zh-headlines"
+
+    def process_results(self, doc, results):
+        gold = doc["gold"]
+
+        return {
+            "avg_f1": (doc["answer"], int(results[0] != "是"), gold, results),
+        }
+
+
+class ZHBigData(StockMovement):
+    DATASET_PATH = "ChanceFocus/flare-zh-bigdata"
+    CHOICE_DICT = {
```

Comment:
Does it mean for Chinese version stockmovement dataset, we still need the model to output english response?